keras_nlp/tokenizers/word_piece_tokenizer.py

```diff
@@ -207,14 +207,25 @@ def __init__(
         self.suffix_indicator = suffix_indicator
         self.oov_token = oov_token
 
-        self._fast_word_piece = tf_text.FastWordpieceTokenizer(
-            vocab=self.vocabulary,
-            token_out_type=self.compute_dtype,
-            suffix_indicator=suffix_indicator,
-            unknown_token=oov_token,
-            no_pretokenization=True,
-            support_detokenization=True,
-        )
+        try:
```

Comment:
Can we just add our own explicit test `if not oov_token in self.vocabulary: raise error...`

If they did things correctly the mask will 99% of the time be an early token in the vocab so it is not really a performance concern.

This will reduce dependency on tf.text errors. E.g. what if they through a different RuntimeError.

Reply:
@mattdangerw This is the concern, if user doesn't provide `oov_token` this if condition might take a very long time.

Reply:
If the user doesn't provide an oov_token, we are in an error condition state. We don't need to be **that** efficient there, we won't even be doing training. We care about efficiency when the code is in the correct state for (say) multiple training runs.

Try it out. Bert has a roughly ~30k vocab, that's fairly standard. I bet you would hardly notice the time it take to iterate and check.

Reply:
Yeah, takes about `25ms` with a vocabulary of size 281132.